internal/kafka/internal/workers/cluster_mgrs/cleanup_clusters_mgr.go

```diff
@@ -95,6 +101,20 @@ func (m *CleanupClustersManager) processCleanupClusters() error {
 	return errList
 }
 
+func (m *CleanupClustersManager) reconcileCleanupEnterpriseCluster(cluster api.Cluster) error {
```

Comment:
Will need to sync this method with the changes in https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1504 to ensure that the certificates are revoked for the given cluster

Reply:
That should probably be a separate PR?

Reply:
Yes. This will be taken care in #1504 

It was more a note for myself to not forget.